ChaosMod/Components/LuaScripts.cpp

```diff
@@ -599,6 +599,8 @@ LuaScripts::ParseScriptReturnReason LuaScripts::ParseScriptRaw(std::string scrip
 		E("GetAllPeds", GetAllPedsArray),
 		E("CreatePoolPed", CreatePoolPed),
 
+		E("TeleportPlayer", [](LuaVector3 pos, bool noOffset) { TeleportPlayer(pos.X, pos.Y, pos.Z, noOffset); }),
```

Comment:
Would this need to be called like `TeleportPlayer(LuaVector3(1.0, 1.0, 1.0), false);`? If so it might be better to just directly take the x,y,z as params.